storyblok/storyblok-design-system
# refactor: datatable components to template syntax

## Pull request type

Jira Link: - 




## How to test this PR


Compare:

- https://blok.ink/?path=/docs/design-system-components-sbdatatable--default
- https://storyblok-design-system-git-refactor-ar-18-bb765c-storyblok-com.vercel.app/?path=/docs/design-system-components-sbdatatable--default

It's also good to check in the app:
```cd storyblok-design-system && git checkout refactor/ar-18-sb-datatable```
Then start the server of the app


## What is the new behavior?



- refactors the datatable to vue components instead of the rendering api
- 
- 

## Other information
